src/Parser.js

```diff
@@ -45,7 +45,13 @@ import ParseError from "./ParseError";
 
 /**
  * An initial function (without its arguments), or an argument to a function.
- * The `result` argument should be a ParseNode.
+ *
+ * @param {ParseNode|string} result It's a string if `isFunction=true` or if
+ *     `result="$"` for switching into math mode; otherwise, it's a ParseNode
+ *     and `isFunction` must be false.
```

Comment:
It's probably worth noting that when it is a string, it should be a function name defined by `defineFunction`, e.g. `"\\frac"`.

Reply:
Done.